### DISCUSSION TITLE
The style sheets are missing when I start Firefly III

### DISCUSSION THREAD

**Discussion by kagkouras** [firefly-iii/firefly-iii#6002] [Firefly III]
hello just installed the script but the style doesnt loading what im missing? i have the latest version

![Screenshot_12](https://user-images.githubusercontent.com/13321214/163492251-22036181-35ea-42e4-98d4-bbb2af268b60.png)
![Screenshot_11](https://user-images.githubusercontent.com/13321214/163465592-72e65310-4f54-48cf-8fe3-9d1a0f2597f8.png)

**Comment by kagkouras** [comment]
nothing?no help?

**Reply by JC5** [reply]
You're not exactly giving us much to go on.

**Comment by kagkouras** [comment]
now i ve got this after a fresh install,is the info enough?

![Screenshot_13](https://user-images.githubusercontent.com/13321214/163628137-cb46eb6d-9657-431e-b272-764b877b910b.png)
![Screenshot_14](https://user-images.githubusercontent.com/13321214/163628156-f71f2a5d-7328-4686-8b42-512b2e362658.png)
![Screenshot_15](https://user-images.githubusercontent.com/13321214/163628173-5215182e-3bcc-4005-8d15-15f86dd58ed2.png)
![Screenshot_16](https://user-images.githubusercontent.com/13321214/163628193-5ece1651-4bb4-4336-b86c-8b163fc48d87.png)

**Reply by JC5** [reply]
Sorry I don't use Windows.
